app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
+
+  def update_likes_counter
+    update(likes_counter: likes.count)
+  end
+end
```

Comment:
- [ ] It would be nice if you put this method in the `like.rb` file 🙃 
- [ ] Also it would be nice to use one of the [Rails Active Callbacks (https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new comment has been saved to `database`, then make the `update_likes_counter` method `private` 😁 
It can be similar to 👇 

```ruby
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:likes_counter)
  end
```